### DISCUSSION TITLE
有木有传递复杂对象的方法 web Component 都只能传简单的变量

### DISCUSSION THREAD

**Discussion by PromiseYo** [hellof2e/quark-design#314] [Q&A]
有木有传递复杂对象的方法 web Component 都只能传简单的变量

**Comment by sanqi-med** [comment]
浏览器原生标签都只能传基本数据类型，如果需要传递复杂对象，有两种方式，一种是我们目前采用的，通过ref拿到组件引用，调用方法传递对象，[参考](https://quarkc.hellobike.com/#/zh-CN/docs/reactive)得高级技巧部分；另外一种是将复杂对象jsonString,当字符串传进去，组件里面在进行jsonParase.
